locale/locale-es-ES.yaml

```diff
@@ -0,0 +1,321 @@
+      add comment:
+
+      phb:
+      spell:
+      spell list:
+      class feature:
+      note:
+      descriptive text box:
+      monster stat block:
+      wide monster stat block:
+      cover page:
+      artist credit:
+
+      tables:
+      class table:
+      half class table:
+      table:
+      wide table:
+      split table:
+
+  styleTab:
+    snippets:
+      styleEditor:
+      removeDropCap:
+      tweakDropCap:
+      addComment:
+
+      print:
+      a4PageSize:
+      squarePageSize:
+      inkFriendly:
+
+  propertiesTab:
+    brew:
+    title: título
+    description: descripción
+    thumbnail: miniatura
+    tags: etiquetas
+    add tag: añadir etiqueta
+    systems: Sistemas
+    language: idioma
+    languageSub: Establece el idioma del documento en el HTML. Puede afectar a la corrección gramatical.
+    theme: tema
+    renderer: renderizador
+    Legacy: Legado
+    v3:
+    legacyDemoLink: haz clic aquí para ver la página demo para el renderizador de legado!
+
+    authors: Autores
+    None: Niguno
+    invited authors: autores invitados
+    invite author: invita a un autor
+    invitedSub1: Los usuarios distinguen entre mayúsculas y minúsculas.
+    invitedSub2: Despues de invitar a un autor, envíale el link para editar el brew. Al abrirlo pueden escoger aceptar o denegar la invitación.
+
+    privacy: Privacidad
+    publish: publicar
+    unpublish: despublicar
+    publishedSub: Los documentos publicados serán públicamente visibles y buscables (eventualmente...)
+    delete: Eliminar
+    delete brew: Eliminar brew
+    onlyAuthorDelete: ¿Estás seguro de que quieres eliminar este brew? Ya que eres el único propietario del brew, se eliminará permanentemente.
+    confirm1: ¿Estas REALMENTE seguro? No podrás recuperar el documento.
+    multipleAuthorDelete: ¿Estás seguro de que quieres eliminar este brew de tu colección? Esto te eliminará como editor, pero otros propietarios aún podrán acceder al documento.
+    confirm2: ¿Estas REALMENTE seguro? Perderás accesos de editor a este documento.
+    themesLegacy: Los temas no están en el Renderizador de Legado
+
+sharePage:
+  metadataWindow:
+    Authors: Autores
+    no authors: No hay autores
+    Systems: Sistemas
+    no systems: Ningún sistema
+    Tags: Etiquetas
+    no tags: Sin etiquetas
+    Description: Descripción
+    No description: Sin descripción
+    Updated: Actualizado
+
+  sourceDropdown:
+    source: Código fuente
+    view: vista previa
+    download: descargar
+    clone to new: clonar como nuevo
+
+accountPage:
+  Account Information: Información de cuenta
+  username: Nombre de usuario
+  noUser: Actualmente no hay ningún usuario con sesión iniciada
+  lastLogin: Último inicio de sesión
+  Homebrewery Information: Información de Homebrewery
+  brewsOnHomebrewery: Brews en Homebrewery
+  Google Information: Información de Google
+  linkedToGoogle: Cuenta vinculada a Google
+  brewsOnGoogleDrive: Brews en Google Drive
```

Comment:
This should just be `brewsOnDrive`, or the accountpage.jsx should have the corresponding part match this `brewsOnGoogleDrive`.  Right now it is not translating.